system/jlib/jstats.cpp

```diff
@@ -2242,7 +2242,7 @@ class CStatisticCollection : public CInterfaceOf<IStatisticCollection>
             ++scopeItem;
         }
 
-        CStatisticCollection * tgtScopeCollection = ensureSubScopePath(path);
+        CStatisticCollection * tgtScopeCollection = ensureSubScopePath(std::move(path));
```

Comment:
More efficient is to change the prototype of ensureSubScopePath:

    CStatisticCollection * ensureSubScopePath(const std::initializer_list<const StatsScopeId> & path)